editor/plugins/org.fusesource.ide.wsdl2rest.ui/src/org/fusesource/ide/wsdl2rest/ui/wizard/pages/WsdlValidator.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/ 
+package org.fusesource.ide.wsdl2rest.ui.wizard.pages;
+
+import java.io.InputStream;
+import java.net.URL;
+import org.apache.commons.httpclient.HttpStatus;
+import org.eclipse.core.databinding.validation.IValidator;
+import org.eclipse.core.databinding.validation.ValidationStatus;
+import org.eclipse.core.runtime.IStatus;
+import org.fusesource.ide.wsdl2rest.ui.internal.UIMessages;
+
+/**
+ * Validate that the WSDL is an accessible file, whether local or remote.
+ * @author brianf
+ */
+public class WsdlValidator implements IValidator {
+
+	/**
+	 * Test to ensure that the URL is accessible.
+	 * @param urlText
+	 * @return
```

Comment:
might worth mentioning that the return value is HttpStatus.SC_OK in case of success and -1 otherwise.
Or to change it by returning a boolean as the name isURLAccessible of the method might better suggest 
https://github.com/jbosstools/jbosstools-fuse/pull/1138#discussion_r180447925